core/src/test/java/com/gcinterceptor/core/GarbageCollectorControlInterceptorTest.java

```diff
@@ -46,8 +46,9 @@ public void testGarbageCollectorControlInterceptorHeapExecutor() throws Interrup
 		assertFalse(heap.hasCollected()); // Since we call before 127 times, but after only 126, there is one "request
 											// remaining" in queue.
 		GCI.after(sr);
-
 		while (GCI.doingGC()) {
+			sr = GCI.before();
```

Comment:
I java, variable names are not capitalized*, couls you rename GCI to gci?

* sorry, but I'm on mobile, but could you please take a look at the google or sun java style guide?

Reply:
Done... Thanks!